### PR TITLE
editor: refactor multimedia plugin

### PR DESCRIPTION
**Pull request by elbotho**
- adds shared renderer for frontend and editor
- simplifies editor code:
   - remove unused important config
   - remove drag&drop resizer ( another 10kb compressed gone from editor routes)
   - only two sizes are allowed (accessible via settings)

ping @LarsTheGlidingSquirrel I think you might need to change `plugins` to `allowedPlugins` in edusharing, sorry ☀️

[Editor-Preview](https://frontend-git-refactor-multimedia-serlo.vercel.app/entity/repository/add-revision/1555)

### CODE REVIEW

**Comment by Entkenntnis** [comment]
looks like you forgot to run ´yarn`, lockfile outdated :)

File: src/serlo-editor/plugins/multimedia/renderer.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx'
+
+export interface MultimediaRendererProps {
+  mediaWidth: number // 25 | 50 | 75 | 100 Percent
```
Comment (LarsTheGlidingSquirrel) [inline src/serlo-editor/plugins/multimedia/renderer.tsx:4]: Is this still relevant if the user can only set two sizes in the settings?

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
Looks way clearer now. Also works in edu-sharing integration. Thank you!

File: src/serlo-editor/plugins/multimedia/renderer.tsx (hunk 1 above)
Comment (elbotho) [reply src/serlo-editor/plugins/multimedia/renderer.tsx:4]: unfortunately yes, since this does not change currently stored values.

**Review by Entkenntnis — APPROVED** [review]
Thank you! Haven't noticed anything off, good to go.

**Pull request merged**